### DISCUSSION TITLE
can you explain what "image_negative" means?

### DISCUSSION THREAD

**Discussion by dingshanliang** [cubiq/ComfyUI_IPAdapter_plus#391] [Q&A] [answered]
In what scenarios should I use  “image_negative"?
Thanks~

**Comment by cubiq** [comment, marked as answer]
Image negative can be used for noise injection.

There's a noise node that you can use to generate noise and send it to the negative image input. That usually grants better results.

Alternatively you can use a negative image exactly like a negative prompt.

**Reply by dingshanliang** [reply]
Thanks for your kind hearted reply.  😊 I've followed your latest video on youtube related with this part. I kind of get it.
